public/js/photoswipe-ui-default.min.js

```diff
@@ -501,7 +501,7 @@
                     name: 'button--info',
                     option: 'infoEl',
                     onTap: function() {
-                        console.log(_captionContainer, 'caption--empty', pswp.currItem.title);
+                        _toggleShareModal();
```

Comment:
Usually, the leading `_` means that the function used is private, or somehow not part of the public API.  Or that it hides functionality that should not be relied upon as a user of the API. Is that the case here?

Reply:
Yes, it's a private function but it's being called from within the library. I edited the library's code. Unfortuantly, there isn't a way to toggle the modal from the public API.

Right now the metadata modal piggy-backs on the share modal provided by the library. If modifying the library's code to toggle the share modal is undesirable, we would have to implement a custom modal.

Reply:
@kokx Do you have any comments?

Reply:
Yeah. That I hate that something like this is necessary. But if it absolutely is necessary, its okay I guess.